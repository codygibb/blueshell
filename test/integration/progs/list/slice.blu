l := [4, 5, 6, 7]
print l[1:3]
print l[:3]
print l[1:]
print l[:-2]
print l[:-4]
print l[1:-4]
