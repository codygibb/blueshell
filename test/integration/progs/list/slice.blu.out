[5, 6]
[4, 5, 6]
[5, 6, 7]
[4, 5]
[]
error@7: Invalid_slice (1, -4)
